src/api/spec/policies/cloud/user/upload_job_policy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe CommentPolicy do
+  let(:uploader) { create(:confirmed_user, login: 'burdenski') }
+  let(:admin_user) { create(:admin_user, login: 'admin') }
+  let(:staff_role) { Role.where(title: 'Staff').first_or_create(global: true) }
+  let(:staff_user) { create(:confirmed_user, login: 'staff', roles: [staff_role]) }
```

Comment:
Might be worth to wrap this into a staff_user factory.

Reply:
I would leave it as it is. I do not see the advantage of having a factory for this here and we will end with a factory for everything. 😖